ping-payload-extensions/extensions/template.md

```diff
@@ -9,7 +9,8 @@ Ping payload
 
 
 [Container Name] = Container(
-  type: [Type Number],
+  enr_seq: uint64,
```

Comment:
I think to clarify we should call this [Container Name] into just Ping message?

Reply:
Like you did here basically: https://github.com/ethereum/portal-network-specs/pull/359/files#diff-3b45b93827ccba3233f47f645ca05b3330998798cfa8180b1764eb69ba52f0dfR38